### ISSUE TITLE
npm peer requirements (for juliancwirko:postcss) not installed

### ISSUE THREAD

**Issue by bartenra** [Meteor-Community-Packages/meteor-postcss#46]
Are the installation instructions in the README up to date?

I get the following warning when building with the specified versions before it fails completely:

```
Building App Bundle Locally
WARNING: npm peer requirements (for juliancwirko:postcss) not installed:    
 - postcss@6.0.22 installed, postcss@8.2.x needed
 - postcss-load-config@1.2.0 installed, postcss-load-config@3.0.x needed
```

**Comment by Saeeed-B** [comment]
@StorytellerCZ hi .
please update requirements .

![image](https://user-images.githubusercontent.com/36485982/127484261-236169c5-4223-4ae3-8f4f-d2a0f4fc4b49.png)

**Comment by StorytellerCZ** [comment]
Should be fixed in 2.0.7
